scripts/ci/pr-bot/processNewPrs.ts

```diff
@@ -45,15 +45,24 @@ import { CheckStatus } from "./shared/checks";
  * (in which case that's all we need to do).
  */
 function needsProcessed(pull: any, prState: typeof Pr): boolean {
-  if (!pull.labels.find((label) => label.name.toLowerCase() === "go")) {
+  if (
+    !pull.labels.find(
+      (label) =>
+        label.name.toLowerCase() === "go" ||
+        label.name.toLowerCase() == "python"
+    )
+  ) {
     console.log(
-      `Skipping PR ${pull.number} because it doesn't contain the go label`
+      `Skipping PR ${pull.number} because it doesn't contain the go or python labels`
     );
     return false;
   }
-  const firstPrToProcess = new Date(2022, 2, 2, 20);
+  const firstPrToProcess = new Date(2022, 7, 16, 14); // June 16 2022, 14:00 UTC (note that Java months are 0 indexed)
   const createdAt = new Date(pull.created_at);
-  if (createdAt < firstPrToProcess) {
+  if (
+    createdAt < firstPrToProcess &&
+    !pull.labels.find((label) => label.name.toLowerCase() === "go")
```

Comment:
do we need to exclude `go` here given that the bot has already been enabled for go? and probably all outstanding `go` prs have been processed already, except for maybe a couple go of PRs that happen to be submitted before two successive iteration of this script?

Reply:
We're excluding go because we don't want it to be skipped ever. Its worth noting that this would retroactively apply to in flight prs, even if the PR bot has already taken some action on them, so without this exclusion this would break the flow on all in-flight PRs. Functionally, the biggest implication would be that if its been assigned to a non-committer and that non-committer approves it won't get auto-routed to a committer